server/models/Request.js

```diff
@@ -1,35 +1,32 @@
 const mongoose = require("mongoose");
 
 const requestSchema = new mongoose.Schema({
-    user_id:{
+    userId:{
         type: String,
         required: true,
       },
-    sitter_id: {
+    sitterId:{
         type: String,
         required: true,
     },
-    start: {
+    start:{
         type: Date,
         required: true
     },
     end:{
         type: Date,
-        required: true
-    },
-    accepted:{
-        type: Boolean,
-        default: False
+        required: true,
     },
-    declined: {
-        type: Boolean,
-        default: False
+    status:{
+        type: String,
+        enum : ["accepted", "declined"],
```

Comment:
I think it's a good idea to differentiate between `pending` and `declined` so let's add `pending` as an accepted value for this and default to it.